docs/source/auth/index.rst

```diff
@@ -0,0 +1,639 @@
+.. _auth:
+
+=====================
+MLflow Authentication
+=====================
+
+MLflow supports basic HTTP authentication to enable access control over experiments and registered models.
+Once enabled, any visitor will be required to login before they can view any resource from the Tracking Server.
+
+.. contents:: Table of Contents
+  :local:
+  :depth: 2
+
+MLflow Authentication provides Python and REST API for managing users and permissions. 
+
+.. toctree::
+  :glob:
+  :maxdepth: 1
+
+  *
+
+Overview
+========
+
+To enable MLflow authentication, launch the MLflow UI with the following command:
+
+.. code-block:: bash
+
+    mlflow server --app-name basic-auth
+
+
+Server admin can choose to disable this feature anytime by restarting the server without the ``app-name`` flag. 
+Any users and permissions created will be persisted on a SQL database and will be back in service once the feature is re-enabled.
+
+Due to the nature of HTTP authentication, it is only supported on a remote Tracking Server, where users send
+requests to the server via REST APIs.
+
+How It Works
+============
+
+Permissions
+-----------
+
+The available permissions are:
+
+.. list-table::
+   :widths: 10 10 10 10 10
+   :header-rows: 1
+
+   * - Permission
+     - Can read
+     - Can update
+     - Can delete
+     - Can manage
+   * - ``READ``
+     - Yes
+     - No
+     - No
+     - No
+   * - ``EDIT``
+     - Yes
+     - Yes
+     - No
+     - No
+   * - ``MANAGE``
+     - Yes
+     - Yes
+     - Yes
+     - Yes
+   * - ``NO_PERMISSIONS``
+     - No
+     - No
+     - No
+     - No
+
+The default permission for all users is ``READ``. It can be changed in the :ref:`configuration <configuration>` file.
+
+Permissions can be set on two MLflow components: Experiments and Registered Models.
+Certain APIs can only be requested if the user has the required permission.
+Otherwise, a "Permission Denied" response will be returned.
+
+.. list-table::
+   :widths: 10 10 10 10
+   :header-rows: 1
+
+   * - API
+     - Endpoint
+     - Method
+     - Permission
+   * - :ref:`Create Experiment <mlflowMlflowServicecreateExperiment>`
+     - ``2.0/mlflow/experiments/create``
+     - ``POST``
+     - None
+   * - :ref:`Get Experiment <mlflowMlflowServicegetExperiment>`
+     - ``2.0/mlflow/experiments/get``
+     - ``GET``
+     - ExperimentPermission.can_read
```

Comment:
If we split the table, can we remove `ExperimentPermission`?

Reply:
i have changed the section a bit